src/osdc/Objecter.cc

```diff
@@ -189,82 +189,82 @@ void Objecter::init()
-    pcb.add_u64_counter(l_osdc_osdop_mapext, "osdop_mapext");
-    pcb.add_u64_counter(l_osdc_osdop_sparse_read, "osdop_sparse_read");
-    pcb.add_u64_counter(l_osdc_osdop_clonerange, "osdop_clonerange");
-    pcb.add_u64_counter(l_osdc_osdop_getxattr, "osdop_getxattr");
-    pcb.add_u64_counter(l_osdc_osdop_setxattr, "osdop_setxattr");
-    pcb.add_u64_counter(l_osdc_osdop_cmpxattr, "osdop_cmpxattr");
-    pcb.add_u64_counter(l_osdc_osdop_rmxattr, "osdop_rmxattr");
-    pcb.add_u64_counter(l_osdc_osdop_resetxattrs, "osdop_resetxattrs");
-    pcb.add_u64_counter(l_osdc_osdop_tmap_up, "osdop_tmap_up");
-    pcb.add_u64_counter(l_osdc_osdop_tmap_put, "osdop_tmap_put");
-    pcb.add_u64_counter(l_osdc_osdop_tmap_get, "osdop_tmap_get");
-    pcb.add_u64_counter(l_osdc_osdop_call, "osdop_call");
-    pcb.add_u64_counter(l_osdc_osdop_watch, "osdop_watch");
-    pcb.add_u64_counter(l_osdc_osdop_notify, "osdop_notify");
-    pcb.add_u64_counter(l_osdc_osdop_src_cmpxattr, "osdop_src_cmpxattr");
+    pcb.add_u64_counter(l_osdc_op_rmw, "op_rmw", "Read-modify-write operations");
+    pcb.add_u64_counter(l_osdc_op_pg, "op_pg", "PG operation, no object");
+
+    pcb.add_u64_counter(l_osdc_osdop_stat, "osdop_stat", "OSD stat operations");
+    pcb.add_u64_counter(l_osdc_osdop_create, "osdop_create", "OSD create object operations");
+    pcb.add_u64_counter(l_osdc_osdop_read, "osdop_read", "OSD read operations");
+    pcb.add_u64_counter(l_osdc_osdop_write, "osdop_write", "OSD write operations");
+    pcb.add_u64_counter(l_osdc_osdop_writefull, "osdop_writefull", "OSD write full object operations");
+    pcb.add_u64_counter(l_osdc_osdop_append, "osdop_append", "OSD append operation");
+    pcb.add_u64_counter(l_osdc_osdop_zero, "osdop_zero", "OSD set object to zero operations");
+    pcb.add_u64_counter(l_osdc_osdop_truncate, "osdop_truncate", "OSD truncate object operations");
+    pcb.add_u64_counter(l_osdc_osdop_delete, "osdop_delete", "OSD delete object operations");
+    pcb.add_u64_counter(l_osdc_osdop_mapext, "osdop_mapext", "OSD map extent operations");
+    pcb.add_u64_counter(l_osdc_osdop_sparse_read, "osdop_sparse_read", "OSD sparse read operations");
+    pcb.add_u64_counter(l_osdc_osdop_clonerange, "osdop_clonerange", "OSD clone range operations");
+    pcb.add_u64_counter(l_osdc_osdop_getxattr, "osdop_getxattr", "OSD get extended attribute operations");
+    pcb.add_u64_counter(l_osdc_osdop_setxattr, "osdop_setxattr", "OSD set extended attribute operations");
+    pcb.add_u64_counter(l_osdc_osdop_cmpxattr, "osdop_cmpxattr", "OSD extended attribute comparison operations");
+    pcb.add_u64_counter(l_osdc_osdop_rmxattr, "osdop_rmxattr", "OSD remove extended attribute operations");
+    pcb.add_u64_counter(l_osdc_osdop_resetxattrs, "osdop_resetxattrs", "OSD reset extended attribute operations");
+    pcb.add_u64_counter(l_osdc_osdop_tmap_up, "osdop_tmap_up", "OSD trivial map update operations");
```

Comment:
I would suggest saying TMAP instead of "trivial map", I think the former is more widely understood